react/ConfirmationMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+
+import { useOrderGroup } from './components/OrderGroupContext'
+
+const CSS_HANDLES = ['confirmationMessage']
+
+const ConfirmationMessage: FC = () => {
+  const handles = useCssHandles(CSS_HANDLES)
+  const orderGroup = useOrderGroup()
+  const profile = orderGroup.orders[0].clientProfileData
+
+  return (
+    <p
+      className={`${handles.confirmationMessage} mt5 t-body tc c-muted-1 lh-copy`}
+    >
+      <FormattedMessage
+        id="store/header.email"
+        values={{
+          lineBreak: <br />,
+          userEmail: <strong className="nowrap">{profile.email}</strong>,
```

Comment:
maybe add a handle here would be nice too

Reply:
I don't think it's needed. It's possible to select it via its parent handle: `.confirmationMessage strong { ... }`